packages/mobile/src/account/SyncBankAccountScreen.tsx

```diff
@@ -49,7 +50,7 @@ const SyncBankAccountScreen = ({ route }: Props) => {
       // TODO(wallet#1447): handle errors from IHL
       return
     }
-    // TODO(wallet#1449): redirect to Bank Account List Page
+    navigate(Screens.BankAccounts, { newPublicToken: publicToken })
```

Comment:
I found out on line 59 syncingBankAccount was not added as a key to the translation file

Reply:
fixed!